tensorboard/uploader/uploader.py

```diff
@@ -581,6 +581,157 @@ def _create_point(self, tag_proto, event, value):
         return point
 
 
+class _TensorBatchedRequestSender(object):
```

Comment:
This class generously copies from _ScalarBatchedRequestSender. David, you mentioned offline that this is probably fine but would like to hear if you have an updated opinion after you've seen the code.  I did refactor out some common non-trivial logic in previous PRs and reuse it here.